src_new/adapters/prebid.js

```diff
@@ -191,18 +191,17 @@ function getPBCodeWithoutWidthAndHeight(divID, adapterID){
 exports.getPBCodeWithoutWidthAndHeight = getPBCodeWithoutWidthAndHeight;
 /* end-test-block */
 
-function generatedKeyCallback(adapterID, adUnits, adapterConfig, impressionID, generatedKey, kgpConsistsWidthAndHeight, currentSlot, keyConfig, currentWidth, currentHeight){
+function generatedKeyCallback(adapterID, adUnits, adapterConfig, impressionID, generatedKey, kgpConsistsWidthAndHeight, currentSlot, keyConfig, currentWidth, currentHeight, banner_sizes){
 
 	var code, sizes, divID = currentSlot.getDivID();
-
+	
 	if(kgpConsistsWidthAndHeight){
 		code = refThis.getPBCodeWithWidthAndHeight(divID, adapterID, currentWidth, currentHeight);
-		sizes = [[currentWidth, currentHeight]];
+		sizes = banner_sizes && banner_sizes.length > 1 ? banner_sizes :  [[currentWidth, currentHeight]];		
```

Comment:
Can banner_sizes be assigned to sizes if its length is 1? Any specific reason we are checking for length above 1?